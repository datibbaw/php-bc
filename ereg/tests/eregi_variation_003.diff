006+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
006- Error: 8192 - Function eregi() is deprecated, %s(71)
016+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
016- Error: 8192 - Function eregi() is deprecated, %s(71)
026+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
026- Error: 8192 - Function eregi() is deprecated, %s(71)
036+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
036- Error: 8192 - Function eregi() is deprecated, %s(71)
046+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
046- Error: 8192 - Function eregi() is deprecated, %s(71)
056+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
056- Error: 8192 - Function eregi() is deprecated, %s(71)
066+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
066- Error: 8192 - Function eregi() is deprecated, %s(71)
076+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
076- Error: 8192 - Function eregi() is deprecated, %s(71)
086+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
086- Error: 8192 - Function eregi() is deprecated, %s(71)
096+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
096- Error: 8192 - Function eregi() is deprecated, %s(71)
106+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
106- Error: 8192 - Function eregi() is deprecated, %s(71)
116+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
116- Error: 8192 - Function eregi() is deprecated, %s(71)
126+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
126- Error: 8192 - Function eregi() is deprecated, %s(71)
136+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
136- Error: 8192 - Function eregi() is deprecated, %s(71)
146+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
146- Error: 8192 - Function eregi() is deprecated, %s(71)
156+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
156- Error: 8192 - Function eregi() is deprecated, %s(71)
166+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
166- Error: 8192 - Function eregi() is deprecated, %s(71)
176+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
176- Error: 8192 - Function eregi() is deprecated, %s(71)
186+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
186- Error: 8192 - Function eregi() is deprecated, %s(71)
197+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
197- Error: 8192 - Function eregi() is deprecated, %s(71)
207+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
207- Error: 8192 - Function eregi() is deprecated, %s(71)
217+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
217- Error: 8192 - Function eregi() is deprecated, %s(71)