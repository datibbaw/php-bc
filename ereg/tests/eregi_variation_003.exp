*** Testing eregi() : usage variations ***
Error: 8 - Undefined variable: undefined_var, %s(61)
Error: 8 - Undefined variable: unset_var, %s(64)

Arg value 0 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 12345 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value -2345 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 10.5 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value -10.5 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 101234567000 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 1.07654321E-9 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 0.5 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value string 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value string 
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}
Error: 4096 - Object of class stdClass could not be converted to string, %s(70)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(71)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}
Done