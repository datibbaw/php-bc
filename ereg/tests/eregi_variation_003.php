<?php
/* Prototype  : proto int eregi(string pattern, string string [, array registers])
 * Description: Regular expression match 
 * Source code: ext/standard/reg.c
 * Alias to functions: 
 */

function test_error_handler($err_no, $err_msg, $filename, $linenum, $vars) {
	echo "Error: $err_no - $err_msg, $filename($linenum)\n";
}
set_error_handler('test_error_handler');

echo "*** Testing eregi() : usage variations ***\n";

// Initialise function arguments not being substituted (if any)
$pattern = 'h(.*)lo!';
$string = 'hello!';

//get an unset variable
$unset_var = 10;
unset ($unset_var);

//array of values to iterate over
$values = array(

      // int data
      0,
      1,
      12345,
      -2345,

      // float data
      10.5,
      -10.5,
      10.1234567e10,
      10.7654321E-10,
      .5,

      // null data
      NULL,
      null,

      // boolean data
      true,
      false,
      TRUE,
      FALSE,

      // empty data
      "",
      '',

      // string data
      "string",
      'string',

      // object data
      new stdclass(),

      // undefined data
      $undefined_var,

      // unset data
      $unset_var,
);

// loop through each element of the array for registers

foreach($values as $value) {
      echo "\nArg value $value \n";
      var_dump( eregi($pattern, $string, $value) );
      var_dump($value);
};

echo "Done";
?>
