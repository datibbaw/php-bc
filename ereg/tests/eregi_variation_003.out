*** Testing eregi() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_003.php(61)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_003.php(64)

Arg value 0 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 12345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value -2345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value -10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 101234567000 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 1.07654321E-9 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 0.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value string 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value string 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_003.php(70)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(6)
array(2) {
  [0]=>
  string(6) "hello!"
  [1]=>
  string(2) "el"
}
Done